lib/ledger_sync/adaptors/netsuite/department/operations/create.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# require_relative '../../operation/create'
+
+module LedgerSync
+  module Adaptors
+    module NetSuite
+      module Department
+        module Operations
+          class Create < NetSuite::Operation::Create
+            class Contract < LedgerSync::Adaptors::Contract
+              params do
+                required(:external_id).value(:string)
```

Comment:
I could have done this wrong, but I think this should be `required(:external_id).filled(:string)` (not `value`.

Reply:
@ryanwjackson I was following what we do for ns `account`, `currency`, `customer` etc, which all use `.value(:string)`

After looking at the [dry-schema docs](https://dry-rb.org/gems/dry-schema/1.4/basics/macros/#value) I think you are right in saying `.filled` is what we should have.

Reply:
We can do a find all and replace probably.  We can do it outside this PR so it doesn't block you.

Reply:
Added to #167